tasks/main.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Create user accounts
+  user:
+    name: "{{ podman_user }}"
+    uid: "{{ podman_user_uid }}"
+
+- name: Configure loginctl to prevent logout causing podman to exit
+  command: loginctl enable-linger {{ podman_user }}
+
+- name: Set podman variable
+  set_fact:
+    podman_conf: podman.conf
```

Comment:
As above, we can check with osbs team whether there is any use cases that using unprivileged ping during building process.